src/components/Employee/Compensation/Compensation.test.tsx

```diff
@@ -81,7 +83,7 @@ describe('Compensation', () => {
       await user.click(continueButton)
 
       await waitFor(() => {
-        expect(screen.getByText('Job title')).toBeInTheDocument()
+        expect(screen.getByTestId('data-view')).toBeInTheDocument()
```

Comment:
Checks on the data-view which is only visible on the list page which is what this test is checking

This was necessary because of differences in how the cards and table render, but this check should do the same thing

Same applies to the other swaps of job title below